tests/Tutor.Infrastructure.Tests/TestData/EventStore/TestEventTypes.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using Tutor.Core.BuildingBlocks.EventSourcing;
+
+namespace Tutor.Infrastructure.Tests.TestData.EventStore
```

Comment:
I'm not sure what to do about CodeFactor failing because of this, I feel like it's better to keep these classes in one file in this specific case.

Reply:
Fixed by wrapping in static class.